DatabaseAndReporting/Integration/02_ManageQueryCategories.md

```diff
@@ -0,0 +1,218 @@
+
+
+## Create a query category
+
+Create a query category for new managed queries you want to assign to a certain category.
+
+#### Prerequisites 
+
+No prerequisites to fulfill.
+
+#### Procedure
+
+*Database and reporting > Settings > Tab QUERY CATEGORIES*
+
+![Query categories](../../Assets/Screenshots/DatabaseAndReporting/Settings/QueryCategories/QueryCategories.png "[Query categories]")
+
+
+1. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.   
+    A plus sign is displayed in the query category column.
+
+    ![Add query category](../../Assets/Screenshots/DatabaseAndReporting/Settings/QueryCategories/AddQueryCategory.png "[Add query category]")
+
+2. Click the plus sign in the query category column.    
+    The *Add element* window is displayed in the query category column.
+
+    ![Add element](../../Assets/Screenshots/DatabaseAndReporting/Settings/QueryCategories/AddElementCategory.png "[Add element]")
+
+3. Enter a name for the query category in the *Name* field.
+
+4. Enter a key for the query category in the *Key* field.  
+
+[comment]: <> (Key wird hier irgendwann abgebaut)
+
+5. Click the [SAVE] button.   
+    The query category has been created. The *Add element* window is closed. The new query category is displayed in the query category column.
+
+    ![Query category created](../../Assets/Screenshots/DatabaseAndReporting/Settings/QueryCategories/QueryCategoryCreated.png "[Query category created]")
+
+    > [Info] To add an additional query category, repeat the steps 1 to 5. To add a query sub-category, see [Create a query sub-category](#create-a-query-sub-category).
+
+6. Click the [SAVE] button in the upper right corner.   
+    The *Submitting data...* view is displayed shortly while saving. 
+
+    ![Submitting data](../../Assets/Screenshots/DatabaseAndReporting/Settings/QueryCategories/SubmittingData.png "[Submitting data]")
+
+    The new query category has been saved to the query categories tree, which is displayed in the *Queries* view.
+    > [Info] It may be necessary to press the **F5** key to initialize the *Core1 Platform* for the changes to be displayed in the *Queries* view. 
+    
+## Edit a query category
+
+After you have created a query category, you can edit it. You can create sub-categories, edit the query category data, and change the order of query categories.
+
+### Create a query sub-category
+
+Add any number of query sub-categories to a query category. Further unlimited sub-category levels can be added. 
+
+The procedure to add a query sub-category to a category and to another sub-category is identical. Just select a sub-category instead of a category and follow the steps described below.
```

Comment:
```suggestion
```
Ich persönlich würde das hier weglassen. Weiß zwar, was du meinst, aber es hört sich doch etwas verschachtelt an. 
Alternativ:
The procedure to add a query sub-category to a category is identical to add another sub-category to a sub-category. Just select a sub-category instead of a category and follow the steps described below.